server/atlassian_connect.go

```diff
@@ -95,6 +96,32 @@ func httpACUninstalled(p *Plugin, w http.ResponseWriter, r *http.Request) (int,
 			errors.New("method " + r.Method + " is not allowed, must be POST")
 	}
 
+	// Notify users we have uninstalled an instance
+	// We're doing this first because there's a chance of getting into a state where the clients think there is an
+	// instance_installed, but the admin has deleted the instance somehow (eg. through the slash commands if we
+	// expose them). If that happens, then we'll error out of this function before we get to the end.
+	// We want to make sure to send the notification to clients that there is no instance_installed.
+	p.API.PublishWebSocketEvent(
```

Comment:
Originally I thought I wanted to send out the notification just in case. Because if any of the other calls failed, there is a good chance the instance wouldn't be in a good state, and I didn't want users to be left with menu options for a jira that isn't connected.

0/5 I think the chances of that are so low that we don't need to worry about it. What do you think? 

For now I'm putting the websocket event after everything (in the uninstall commands in command.go).

Reply:
I'm fine with that - ", there is a good chance the instance wouldn't be in a good state, "